### PR TITLE
Fix [#1647]

### PR DESCRIPTION
**Pull request by ksmohamed**

### CODE REVIEW

File: android/src/main/java/com/mr/flutter/plugin/filepicker/FilePickerPlugin.java [hunk 1]
```diff
@@ -23,6 +23,7 @@
 import io.flutter.plugin.common.EventChannel;
 import io.flutter.plugin.common.MethodCall;
 import io.flutter.plugin.common.MethodChannel;
+import io.flutter.plugin.common.PluginRegistry.Registrar;
```
Comment (MrCsabaToth) [inline android/src/main/java/com/mr/flutter/plugin/filepicker/FilePickerPlugin.java:26]: Why would you bring V1 back even more? It's deprecated for good.
Correct solution: https://github.com/miguelpruivo/flutter_file_picker/pull/1644

**Review by MrCsabaToth — CHANGES_REQUESTED** [review]

File: android/src/main/java/com/mr/flutter/plugin/filepicker/FilePickerPlugin.java (hunk 1 above)
Comment (ksmohamed) [reply android/src/main/java/com/mr/flutter/plugin/filepicker/FilePickerPlugin.java:26]: This package are missing so its showing error now its clear

**Comment by ksmohamed** [comment]
bug fixed

**Comment by JakobLichterfeld** [comment]
Please drop as #1648 is real solution

**Review by JakobLichterfeld — CHANGES_REQUESTED** [review]
Please drop, real solution is #1648 
btw. the PR title must be self-explanatory

**Pull request closed without merging**